src/registrar/templates/domain_detail.html

```diff
@@ -17,9 +17,10 @@
           <span class="text-bold text-primary-darker">
             Status:
           </span>
-            {% if domain.is_expired %}
+            {# UNKNOWN domains would not have an expiration date and thus would show 'Expired' #}
+            {% if domain.is_expired and domain.state != domain.State.UNKNOWN %}
               Expired
-            {% elif domain.state == domain.State.UNKNOWN or domain.state == domain.State.DNS_NEEDED%}
+            {% elif domain.state == domain.State.UNKNOWN or domain.state == domain.State.DNS_NEEDED %}
```

Comment:
**question (non-blocking)**: A designer can probably help us, but is "DNS needed" the right thing to display here if a domain is in our system but it has never been visited with EPP so it shows up as "UNKNOWN"?

Reply:
Good question. I want to say this has been debated already but can't remember which ticket had an AC to display DNS Needed for Unknown domains. Pretty sure it came from an AC.

Reply:
That was a request from them actually. They didn't want it to show as Unknown or Unset or whatever. They said user won't care, show the same label basically